github_changelog_generator.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency("github_api", ["~> 0.12"])
   spec.add_runtime_dependency("colorize", ["~> 0.7"])
+  spec.add_runtime_dependency("sqlite3")
```

Comment:
DBM is giving me permission denied errors; haven't yet been able to trace the cause, as they don't happen always.

VCR won't help since it doesn't send out Etags, it just assumes that whatever you get is valid for a period you choose beforehand.

So far, SQLite is the only thing that has worked for me both reliably and fast.